templates/cmd/main.go.tpl

```diff
@@ -42,6 +42,11 @@ import (
 
 // main is the entrypoint for the {{ .Config.Name }} service.
 func main() { //nolint: funlen // Why: We can't dwindle this down anymore without adding complexity.
+  exitCode := 1
+  defer func() {
+    os.Exit(exitCode)
+  }
```

Comment:
```suggestion
  defer os.Exit(exitCode)
```

I think this is ok, right @george-e-shaw-iv?

Reply:
Yeah